website/static/js/markdown.js

```diff
@@ -30,12 +31,18 @@ var bootstrapTable = function(md) {
 var oldMarkdownList = function(md) {
     md.block.ruler.after('hr', 'pyMarkdownList', pymarkdownList);
 };
+var mfrURL = window.contextVars.mfrURL;
+var osfURL = window.contextVars.osfURL;
 
 // Full markdown renderer for views / wiki pages / pauses between typing
 var markdown = new MarkdownIt('commonmark', {
     highlight: highlighter,
     linkify: true
-})
+    }).use(require('markdown-it-mfr'), {
+        type: 'osf',
+        pattern: /^http(?:s?):\/\/(?:www\.)?[a-zA-Z0-9 .:]{1,}\/render\?url=http(?:s?):\/\/[a-zA-Z0-9 .:]{1,}\/([a-zA-Z0-9]{5})\/\?action=download|(^[a-zA-Z0-9]{5}$)/,
```

Comment:
When trying to embed a file that doesn't exist, whose guid has five-characters,` @[osf](12345)`, you see a NOT FOUND message like you do in prod.  However, if your incorrect guid has more or less than 5 characters @[osf](1234), it doesn't attempt to make a request to mfr, it just renders it as `@osf`, where the osf is a link, and when clicked, leads to `/<node_id>/wiki/<wiki%20name>/<4 char incorrect guid>/`. This deviates from prod behavior where a 4 or 6 character guid would be requested from MFR, and a Not Found would be displayed.

Reply:
I actually think this is a little better because if the guid is wrong it fails imeddiately instead of loading MFR just to say you got the guid wrong.

It's difficult to fix the @osf issue because the valid markdown for linking is `[label](link)` so when the guid isn't valid it veers into making it a link with a @ before the label. Unsure what to do about that, it's sort of a product question.

Reply:
I agree it's better to not make a request to MFR if we don't have to but are we 100% sure that this 5-char guid assumption always holds? I seem to remember that working on data migrations in the past, on prod, sometimes old stuff doesn't have 5 chars? Could be misremembering or this might not be true for file guids.

Reply:
Also, can you run this by Product @Johnetordoff, and make sure that they are fine with this new behavior?

Reply:
So I thought about this some more and I think your initial thoughts are correct. Having testing it using >5 character guids, it returns a "NOT FOUND" message quickly enough that it's actually more helpful than a dead link, so I'm just going to update the regex.

Reply:
Sounds good @Johnetordoff

Reply:
> Edit: Also +5 charecter guids are forward compatible, so I actually think it's not really a product thing.

@Johnetordoff Do you mean that the current code handles >5-character GUIDS? It looks like the regex still only captures 5 characters (`(^[a-zA-Z0-9]{5}$)`). Or am I misunderstanding?